dashboard/app/views/levels/_teacher.html.haml

```diff
@@ -73,41 +73,18 @@
                     = t('teacher.panel.example')
           = link_to build_script_level_path(@script_level, @section? {section_id: @section.id} : {}) do
             %button.btn.btn-info= t('teacher.panel.try_it_yourself')
-        - if @section
-          .section
-            %h4
-              = t('teacher.panel.section')
-              = link_to @section.name, teacher_dashboard_section_progress_url(@section)
-        - if current_user.sections.many?
+        - if current_user.sections.any?
+          #check-mark-container{style: "display: none"}
+            = check_mark_html
           -# This is rendered into by react
           #teacher-panel-sections
           - if @section && @script_level.stage.lockable?
             %div{:style => "margin: 10px"}
               #stage-locked-text
               %div
               = link_to t('teacher.panel.update_lock_settings'), script_path(@script_level.script)
-      - if @section
-        .scrollable-wrapper
-          %table.section-students
-            - @section.students.order(:name).each do |student|
-              - active = 'active' if @user && student.id == @user.id
-              %tr.section-student{class: active}
-                - user_level = student.last_attempt_for_any(@script_level.levels)
-                - status = activity_css_class(user_level)
-                %td
-                  .level_link{class: status}
-                    - if (user_level.try(:level) || @script_level.oldest_active_level).unplugged?
-                      %span.puzzle-named
-                        Unplugged
-                    - elsif status == 'passed' || status == 'perfect'
-                      = check_mark_html
-                    - else
-                      %span.puzzle-number
-                        = @script_level.position
-                %td.name{class: user_level && user_level.navigator? && !user_level.driver? ? 'navigator' : ''}
-                  = link_to build_script_level_path(@script_level, user_id: student.id, section_id: @section.id) do
-                    = student.name
-                    - if user_level && (user_level.navigator_user_levels.present? || user_level.driver_user_levels.present?) # I am a driver
-                      %i.fa.fa-users.pair-programming-icon
+      - students_info = get_students_info(@script_level)
+      #students_info{data: { students_info: students_info.to_json}}
```

Comment:
This might also be a little weird and there might be a better way to accomplish.

Essentially, we have a chunk of JSON we want to transfer from dashboard to JS. I took an approach similar to https://github.com/code-dot-org/code-dot-org/pull/11322 of passing it via the DOM. It arguably makes more sense there than here. Another option would be to just stick it on `window` or `window.dashboard` somewhere.